apps/heureka/README.md

```diff
@@ -7,11 +7,30 @@ This mircro frontent uses the Heureka POC API which aims to:
 
 - Track the overall state of our technology landscape
 - Establish a unified & complete Patch Management Process:
+
   - Maintain & track vulnerabilities and affected components
   - Document the remediation, classification, and impact of vulnerabilities
   - Document the changes corresponding with patching of vulnerabilities, as well as updating components
 
-# Usage
+  **Coming soon:**
+
+- Configurable initial filters to automatically apply at app startup.
+- Pagination for the Component Instances list in the service details panel.
+
+For your more info Heureka is a Security Posture Management tool designed to manage security issues in a cloud operating system.
+
+Its primary focus is remediation management of security issues such as vulnerabilities, security events, and policy violations while ensuring compliance and audit-ability. The BE code for Heureka including its used API here for frontend is availble in the following repo: https://github.com/cloudoperators/heureka
+
+## Screenshots
+
+IssueMatches list view:
+![Issue Matches List](../assets/images/heureka_issues_matches_list.png)
```

Comment:
<img width="1209" alt="image" src="https://github.com/user-attachments/assets/0737537f-d0cb-480f-af3c-ace69e4c19a3">
Looks like screenshots are not rendered correctly could you double check?